### ISSUE TITLE
Queries stretching lon 180 degrees returns only nodes

### ISSUE THREAD

**Issue by plepe** [drolbr/Overpass-API#523]
```
[out:json][bbox:50,175,55,180];
nwr[place=island];
out body geom qt;
```
This query returns 1 node, 4 ways and 6 relations.

If I extend the bounding box to `50,175,55,-175`, the result only includes nodes - the one node returned in the query above and another at lon=-177.94.

I would expect the query to include all objects returned from the first query and additionally all islands (ways, relations) east of lon=-180.

**Comment by drolbr** [comment]
Thank you, this looks like a bug.

**Comment by drolbr** [comment]
Fixed in 500c5e9fd9d75fdb1b21cbd33c87a419d45f75f1
